test/e2e/framework/fixture.go

```diff
@@ -79,6 +79,7 @@ func TestServerArgs() []string {
 func TestServerArgsWithTokenAuthFile(tokenAuthFile string) []string {
 	return []string{
 		"--discovery-poll-interval=5s",
+		"-v=4",
```

Comment:
This is extremely useful.

Reply:
or harmful as there is lots of noise. What did you miss before?